tools/phpcs-new-rule.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -eo pipefail
+
+array=()
+while IFS=  read -r -d $'\0'; do
+    array+=("$REPLY")
+done < <(find projects -name "*.php" -type f -print0)
```

Comment:
This is going to try to check everything in projects' vendor and wordpress directories too. The `--ignore` below skips actually checking them, but it's still going to waste time listing them here and echoing them on line 11.

Doing `git ls-files -z '*.php'` would avoid that problem, but we'd still be linting files that are supposed to be being ignored (via `.phpcsignore`).

If we want to avoid that too, IMO the best solution would be to just update our standard to include the new rule first and then `composer phpcs:lint -- --sniffs=Whatever.New.Sniff` to test it instead of doing a script at all.

Reply:
After working on this a bit, I realized running phpcs worked fine except I had to increase the memory limit for the CLI on my client. I think probably drop the script completely, but wanted to include it just in case there was something I was missing with the phpcs + setting the memory higher.